repos/system_upgrade/el8toel9/actors/checkdeprecatedrpmsignature/tests/unit_test_checkdeprecatedrpmsignature.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+# from leapp import reporting
```

Comment:
Please remove the comments when you are ready...

Reply:
I would like to keep these as it is the first thing people will do when they starte to write tests for this actor.